components/steering_wheel/include/IO/IO_Digital.h

```diff
@@ -0,0 +1,58 @@
+/**
+ * IO_Digital.h
+ * Digital IO Module Header
+ * @note To transfer this module to another system, the digitalInput_E and the
+ *      typedef'd IO_Digital_Flag_t must be modified to be congruent with the new system
+ */
+
+#pragma once
+
+/******************************************************************************
+ *                             I N C L U D E S
+ ******************************************************************************/
+
+#include "Types.h"
+#include "stm32f103xb.h" /**< Needed for GPIO_TypeDef */
+
+// System Includes
+#include <assert.h>
+
+
+/******************************************************************************
+ *                             T Y P E D E F S
+ ******************************************************************************/
+
+typedef enum
+{
+    SW1 = 0U,
+    SW2,
+    SW4,
+    SW5,
+    BTN_TOP_LEFT,
+    BTN_MIDDLE,
+    BTN_TOP_RIGHT,
+    NUM_INPUTS
+} digitalInput_E;
+
+typedef uint8_t IO_Digital_Flag_t; /**< Current Implementation can handle upto 64 inputs */
```

Comment:
Is this comment accurate? Seems like a uint8 could only handle 8 inputs?

Reply:
No I had not updated when making the edit. Will fix this thanks for pointing out